### PR TITLE
Feature list

### PR DESCRIPTION
**Pull request by pierr**

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import React from 'react';
+window.React = React;
```
Comment (TomGallon) [inline app/index.js]: Why do you expose REACT In Windows ?

File: app/stores/country-list.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import ListStore from 'focus-core/store/list';
+export default new ListStore({identifier: 'countryList'});
```
Comment (TomGallon) [inline app/stores/country-list.js:2]: Property is named différently than its value... That not helps understanding or debbuging. No ?

File: app/index.js (hunk 1 above)
Comment (pierr) [inline app/index.js]: Was here to fix a missing React , I forgot to remove it.

File: app/stores/country-list.js (hunk 2 above)
Comment (pierr) [inline app/stores/country-list.js:2]: CountryList? There is no property name in this case:

``` javascript
import ListStore from 'focus-core/store/list';
const countryListStore =  new ListStore({identifier: 'countryList'});
export default countryListStore ;
```

Would it be better for you?

**Pull request merged**